.github/workflows/build.yml

```
@@ -0,0 +1,25 @@
+name: Build mod
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+    name: Build mod
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'adopt'
+        java-version: '8'
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: ./gradlew -Pnet.minecraftforge.gradle.disableUpdateChecker=true build
+    - name: Upload artifacts
```

```suggestion
    - name: Rename jar
      run: mv build/libs/morerefinedstorage-*.jar build/libs/morerefinedstorage-${GITHUB_SHA::7}.jar
    - name: Upload artifacts
```
This avoids confusion when downloading multiple of these